.github/pull_request_template.md

```diff
@@ -1,37 +1,48 @@
-Fixes | Closes | Resolves #
+# Description
 
-> Anything marked as optional that you didn't need to fill in, please remove it from the PR description. Choose one of the keywords above to refer to the issue this PR solves, followed by the issue number (e.g Fixes # 666). If there is no issue, remove the line. Remove this note after reading.
+>Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. 
+>**THIS WILL BE USED IN RELEASE NOTES**
 
-## Changes:
--
+Description here
 
-## Types of changes
+Fixes # (issue)
 
-What types of changes does your code introduce?
-_Put an `x` in the boxes that apply_
+## Type of change
 
-- [ ] Bugfix (non-breaking change which fixes an issue)
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] Documentation Update
-- [ ] Code style update (formatting, renaming)
-- [ ] Refactoring (no functional changes, no api changes)
-- [ ] Build related changes
-- [ ] Other (please describe): 
+- [ ] Refactoring (includes code styling changes etc)
+- [ ] This change requires a documentation update
+	- If marked - put a docs PR link here
+
+# How Has This Been Tested?

```

Comment:
I do not know how i feel about this section. maybe a bit excessive. I know it is needed for big changes but sometime the changes are so small that not even unit tests are needed.

Reply:
@smartprogrammer93 
This may not apply to all of our PRs but i strongly believe, that 99% of our changes are a things that can be somehow "dev-tested" and this should appear here. This is just to avoid pushing anything not tested.
If PR needs more attention by QA then we can just suggest bigger things which are needed to be tested (hint for QA + additional check by DEV which may result with "OH! I forgot to cover this scenario" while writing to QA about test cases)